canarytokens/models.py

```diff
@@ -1318,14 +1318,24 @@ def get_additional_data_for_notification(self) -> Dict[str, Any]:
                 "token_type",
             ),
         )
-        if "additional_info" in additional_info:
-            additional_info.update(**additional_info.pop("additional_info"))
+        if "additional_data" in additional_data:
+            additional_data.update(**additional_data.pop("additional_data"))
         for key, replacement in [("l", "location"), ("r", "referer")]:
-            if key in additional_info:
-                additional_info[replacement] = additional_info.pop(key)
+            if key in additional_data:
+                additional_data[replacement] = additional_data.pop(key)
             if self.src_data and key in self.src_data:
                 self.src_data[replacement] = self.src_data[key]
-        return additional_info
+
+        if additional_data.get("geo_info", {}).get("country") is not None:
+            additional_data["geo_info"]["continent"] = get_src_ip_continent(
+                self.geo_info.country
```

Comment:
we check if it's in `additional_data` but then query the result on the class? can we make this addtional_data?

Reply:
Yup that doesn't make sense. I'll change it.